pyscal/scalrecommendation.py

```diff
@@ -81,6 +87,7 @@ def __init__(self, low, base, high, tag=None, h=0.01):
                 swcr=low["swcr"],
                 h=h,
                 tag=tag,
+                fast=fast,
```

Comment:
These code lines are deprecated and will be removed, see #228

Reply:
Ah. Did not catch that one, sorry. So the fast variable will not really be needed in the constructor here at all then, only in the interpolate function?

Reply:
Just leave out 'fast=fast' for the situation when the class input is dicts, then it will not conflict with #228..